server/mvcc/backend/hooks_test.go

```diff
@@ -0,0 +1,62 @@
+// Copyright 2021 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package backend_test
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+	"go.etcd.io/etcd/server/v3/mvcc/backend"
+	betesting "go.etcd.io/etcd/server/v3/mvcc/backend/testing"
+)
+
+func TestBackendPreCommitHook(t *testing.T) {
+	bucket := []byte("bucket")
+	key := []byte("key")
+
+	b, _ := betesting.NewTmpBackendCfg(t,
```

Comment:
Could we name it `be` instead of `b`? It seems that this is common name for backend and `b` is suggesting `testing.B`

Reply:
done